source/clear-linux/guides/maintenance/Telemetry-Enable-Disable.rst

```diff
@@ -61,29 +61,28 @@ To disable the telemetry daemon run the following command:
 Opt-out of telemetry
 ********************
 
-If you no longer want to send telemetrics data from your system, you can
-opt-out of the service by running the command:
+To stop sending telemetrics data from your system, opt out of the
+telemetry service:
 
 .. code-block:: console
 
 	sudo telemctl stop
-	sudo telemctl opt-out

```

Comment:
@tpleavitt Did you mean to remove the `sudo telemctl stop` instead? If you just run `sudo telemctl stop`, you will not be opted out of telemetry.